app/src/main/java/com/xmartlabs/gong/AppBase.kt

```diff
@@ -50,9 +50,7 @@ abstract class AppBase : Application() {
                 detectLeakedClosableObjects()
                 detectLeakedRegistrationObjects()
                 detectLeakedSqlLiteObjects()
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-                    detectCleartextNetwork()
-                }
+                detectCleartextNetwork()
```

Comment:
```suggestion
                detectClearTextNetwork()
```

Reply:
It's a StrictMode method, so I can not change it!